packages/api/src/command/internal/get-hie-enabled-feature-flags-status.ts

```diff
@@ -0,0 +1,14 @@
+import { getFeatureFlags, CxFeatureFlagStatus } from "@metriport/core/external/aws/app-config";
+import { Config } from "../../shared/config";
+
+export async function getHieEnabledFFStatus(cxId: string): Promise<CxFeatureFlagStatus> {
+  const region = Config.getAWSRegion();
+  const appId = Config.getAppConfigAppId();
+  const configId = Config.getAppConfigConfigId();
+  const envName = Config.getEnvType();
+  const featureFlags = await getFeatureFlags(region, appId, configId, envName);
+  return {
+    cxsWithCWFeatureFlag: featureFlags.cxsWithCWFeatureFlag.values.includes(cxId),
+    cxsWithCQDirectFeatureFlag: featureFlags.cxsWithCQDirectFeatureFlag.values.includes(cxId),
```

Comment:
I know this is the name of the FF, but it reads very weird. Maybe we just use `cwEnabled` and `cqEnabled` as we're doing on `updateHieEnabledFFs()`?

Reply:
Done.